ebi_eva_common_pyutils/assembly/assembly.py

```diff
@@ -29,3 +31,39 @@ def get_supported_asm_from_ensembl(tax_id: int) -> str:
     if assembly_accession_attribute in response:
         return str(response.get(assembly_accession_attribute))
     return None
+
+
+@lru_cache
```

Comment:
Not sure how this works for cases where the function has no arguments and Python can't compare the input with a previous input... does Python cache the result unconditionally?

Reply:
Yes exactly, it's kind of a lazy shorthand for "just call this method once"... I added `maxsize=None` as I read somewhere it performs a bit better for this use case.